### PR TITLE
Remove child_health tableau data source

### PR DESCRIPTION
**Pull request by emord**
This removes the final dependencies on child_health_monthly_tableau UCR datasource. It intentionally does not remove the data source yet, as I'd like to have this deployed for a week or so, just in case a revert is required. Once this has been live for at least one week, I will clean up all the references to it and delete the table.

I suggest going through commit by commit. The +/- is large, but that is mostly reconciling differences in our test data and the changes to real code are quite small (+117/-76). I believe that I went through each commit and commented on the data changes with what changed and why it changed.

### CODE REVIEW

**Comment by emord** [comment]
@calellowitz for dashboard @jmtroth0 code buddy

(i'm explicitly ignoring stickler for the long lines because I think it's slightly more readable to have that info on one line)

File: custom/icds_reports/utils/aggregation.py [hunk 1]
```diff
@@ -1189,7 +1189,7 @@ def aggregation_query(self):
             LEFT OUTER JOIN "{agg_gm_table}" gm ON child_health.doc_id = gm.case_id AND gm.month = %(start_date)s
             LEFT OUTER JOIN "{agg_pnc_table}" pnc ON child_health.doc_id = pnc.case_id AND pnc.month = %(start_date)s
             LEFT OUTER JOIN "{agg_df_table}" df ON child_health.doc_id = df.case_id AND df.month = %(start_date)s
-            WHERE child_health.doc_id IS NOT NULL AND (child_health.closed_on IS NULL OR child_health.closed_on > %(next_month)s) AND child_health.opened_on < %(next_month)s
```
Comment (calellowitz) [inline custom/icds_reports/utils/aggregation.py]: why can you get rid of this

**Review by calellowitz — APPROVED** [review]
two small questions but unbelievably excited for this

File: custom/icds_reports/utils/aggregation.py [hunk 2]
```diff
@@ -962,6 +962,8 @@ def aggregation_query(self):
         alive_in_month = "(child_health.date_death IS NULL OR child_health.date_death - {} >= 0)".format(start_month_string)
         seeking_services = "(child_health.is_availing = 1 AND child_health.is_migrated = 0)"
         born_in_month = "({} AND child_health.dob BETWEEN {} AND {})".format(seeking_services, start_month_string, end_month_string)
+        valid_in_month = "({} AND {} AND {} AND {} <= 72)".format(open_in_month, alive_in_month, seeking_services, age_in_months)
+        #fully_immunized_eligible = "{} AND {} > 365".format(valid_in_month, age_in_days)
```
[the remark this answers is not in the document]
Comment (emord) [reply custom/icds_reports/utils/aggregation.py]: that doesn't exist anymore https://github.com/dimagi/commcare-hq/pull/21750/commits/485ad68145ac0bb28ce9fa69881b7968dd8eb6a1#diff-41aa77149c22ba130cf8f239d8b26a5fL966 I had started it. had an error and commented it out. In that commit I added it back with the correct calculation

**Pull request merged**